test/tools/libtesteth/ImportTest.cpp

```diff
@@ -239,6 +239,9 @@ void ImportTest::parseJsonStrValueIntoSet(json_spirit::mValue const& _json, set<
 bool ImportTest::findExpectSectionForTransaction(
     transactionToExecute const& _tr, eth::Network const& _net, bool _isFilling) const
 {
+    if (!_isFilling && !m_testInputObject.at("post").get_obj().count(netIdToString(_net)))
```

Comment:
I have no idea what it does :)

Reply:
so the issue was that this function was assuming that filled tests have all the networks, while tests could have only one or a set of networks. out of range happend on json object. 
now it return that post section not found for provided singlenet if there is no network in test post section. still results in an error but not in exception

Reply:
I think that deserves a comment in the code.